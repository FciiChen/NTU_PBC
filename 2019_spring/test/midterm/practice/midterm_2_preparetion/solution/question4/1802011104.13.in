鳳知微猶豫，不過一瞬間。

　　隨即她向前。

　　掠前三步，手指一彈，一抹烏光掠過，啪的落在藤蔓掩映的洞口，在幽邃的深洞激起裊裊回聲。

　　「什麼人！」

　　暴喝立即炸起，夜色中從各個方向飛起無數條人影，直奔洞口而來。

　　暗哨被驚動，注意力齊齊轉向入口，鳳知微毫不猶豫，後退。

　　她向前的身子在原地一個流水般的大逆轉，腳跟一移轉眼間已經暴退三丈，再下一個轉身她已經離開了入口。

　　趁著守衛全部被吸引到入口，她向後而行，這是最紛亂的一瞬間，前莊守衛心無旁騖奔向入口，後莊守衛還沒有得到消息，她在這個夾縫中穿行，不需要再太過小心的躲守衛，以最快的速度奔向碧照廳方向。

　　奔到離碧照廳還有兩個院子的時候，她沉思了一下，停住腳步，伏下身子。


 
　　果然，她身子剛剛低下去，頭頂上就傳來衣袂帶風聲，來者銀色衣袍在藏青蒼穹裡劃出硬朗的弧線，遠遠的也像一抹冷月光，自頭頂一拋而過。

　　金羽衛指揮使。

　　鳳知微無聲舒了口氣，她驚動門口守衛，一方面是提醒在山莊之外潛伏等待接應的寧澄，一方面是吸引暗哨好讓自己快速返回山莊內，還有個最主要的目的，就是要把這個金羽衛指揮使給引出碧照廳。

　　這人像一條潛伏在夜色草叢裡的銀環蛇，讓她不安，想到要在這樣的人目光底下再次潛入碧照廳，她直覺危險，讓寧弈呆在這人身邊和二皇子應對，她也不安，以她對這位指揮使職業的瞭解，只要他知道前院入口有異動，肯定會親身前往查看。

　　果然對方中了她調虎離山之計。

　　鳳知微心裡一口氣還沒舒完，半空中金羽衛指揮使突然回頭。

　　他明明已經掠出很遠，再無可能發現鳳知微的蹤跡，偏偏就在那麼遙遠的距離，驀然回首。

　　月光照著他的臉，僵木而無表情，也像戴了面具，眉目都隱在光源深處，像幾個深不見底的洞。

　　鳳知微一動不動，並沒有刻意抑制呼吸試圖躲藏，而是將自己的氣息收斂意念放空，渾然在這山風凜冽的春夜裡。


 
　　她不相信是自己蹤跡被發現。

　　她知道這只是指揮使多年生涯練就的直覺。

　　有一種人因為潛行黑暗，特別敏銳，聽得見他人內心的聲音。

　　朦朧月色下金羽衛指揮使一動不動立在樹梢，在細細的梢尖載沉載浮，月光被樹枝割成千萬條，疏落在前方的道路上。

　　良久之後，他慢慢掠了掠衣袍，眼神裡掠過一絲疑問，隨即轉身掉頭而去。

　　他的身影如流星一擲千里瞬間消逝。

　　鳳知微還是沒動，連鬆口氣都沒有。

　　不過剎那間。

　　風聲又一響。


 
　　方才還遠遠離開的指揮使，突然再次在路的盡頭出現。

　　他這回在花間小徑上站下，鷹隼般的眼睛四處梭巡，發現還是毫無動靜，才默然離去。

　　鳳知微又等了一會，從地上爬起。

　　剛才和金羽衛指揮使一場關於耐性和定力的較量，好歹沒輸。

　　她一旦起身再不猶豫，順著路奔了幾步，眼看前方有人過來，一閃身躲到一株樹後。

　　這一躲，她突然發現樹後便是深谷，這山莊依山勢而建，處處有絕崖，如果此刻沿這樹爬下去，過一條窄谷，對面就是碧照廳。

　　碧照廳那種設計和位置，是別想從崖下爬上去的，但是給客人安排的客房，在碧照廳上方突出的橫崖之上，連接著這邊一道山梁，可以爬上去，也可以攀援而下偷聽。

　　那裡無法佈置守衛，而且峭壁嶙峋，山崖濕滑，兩山之間山風猛烈，稍不注意便會被吹下去，一旦被發現，對方動動手指便可以將人置之死地。

　　而且在靠近碧照廳附近的崖壁上，都有燈，誰要被接近，一眼就能看見。


 
　　鳳知微也只猶豫了一瞬間。

　　隨即她輕若鴻羽一般，從樹上飄了下去，哧溜一下滑到底，身已在懸崖之上。

　　一手攀住崖壁，一手從髮髻裡拔出一個菱形翡翠壓髮，手指用力，外面那層假翡翠裂開，露出精光熠熠的小匕首。

　　她叼著匕首一路下行，下到一半時停住，那裡是個山隙，約有丈許距離，縱過去，便可從崖後翻到山莊用來休息的客房，她剛才便是從那裡偷偷溜走的。

　　只是這說起來簡單，做起來卻不容易，崖壁上一盞燈在風中滴溜溜轉著，有人守在燈前。

　　鳳知微沉思了一會，滅燈容易，但是必然驚動守衛，怎樣做才最妥帖？

　　山崖下忽然起了一陣風。

　　鳳知微立刻一抬手，一截樹枝破空而上，撞在燈籠鏈條上，並沒有打斷鏈條，卻有一股巧勁，使燈籠猛烈搖晃旋轉，燭火顫然欲熄。

　　兩個守燈侍衛今晚得了命令，燈一刻也不得熄，看見燈顫得厲害，急忙上來護住，一人笑道：「今晚這風好大，平日裡這燈吹不動的。」另一人道：「怕是要熄，先吊上來護住。」


 
　　兩人探身將吊在崖壁上的燈拽上來。

　　身子一錯燈光一暗之時。

　　鳳知微一拽崖上籐條，身子一蕩，衣袂在鐵黑的崖壁上劃破浮游的濕雲，如電光一閃，橫空渡越。

　　不過是燈光一暗又明。

　　她已經貼在對面崖壁。

　　崖上兩名守衛擎著燈，等這陣風過去，其中一人突然往崖下探了探，「咦」了一聲道：「剛才好像看見什麼影子一閃？」

　　「你眼花了吧？」另一人笑道，「誰找死，敢從這裡爬上來？」

　　「我。」

　　清清淡淡的回答驚得兩人一顫，愕然回身，卻什麼都沒看見。


 
　　兩人的腦海裡瞬間掠過「山精鬼魅！」之類的字眼，這個念頭還沒完全閃現，突然覺得喉嚨一緊，一涼。

　　像冬日裡一把雪突兀的塞在了咽喉。

　　帶走了一生裡所有的熱氣。

　　兩人捂著咽喉，發出格格的聲音，站在他們身後的鳳知微，漠然的鬆開勒住他們脖子的胳臂，小心的拎著他們的後衣領，不讓屍體落地發出聲響。

　　隨即快速的剝下一個較瘦守衛的外衣自己穿上，將兩具屍體繼續扶坐在崖邊燈前，將那燈調整了個位置。

　　特製的具有穿透力的光，映在上方崖壁，和頂頭碧照廳的探燈的燈光交織在一起，原本清晰的光照，反而開始模糊。

　　燈下黑。

　　鳳知微原本沒打算冒險上這崖殺守衛，卻在看見那燈的時候改變了主意。

　　燈光如果用得不對，一樣照不出東西來的。
BREAK