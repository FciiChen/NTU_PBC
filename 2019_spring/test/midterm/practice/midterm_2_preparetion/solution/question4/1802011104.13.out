鳳_微 14
碧_廳 9
指_使 8
崖_上 6
羽_指 5
衛_揮 5
金_衛 5
一_間 3
爬_去 3
一_不 2
